exo-setup/src/app-setup.ls

```diff
@@ -27,15 +27,16 @@ class AppSetup extends EventEmitter
         @services.push do
             role: service-role
             location: service-data.location
-            docker-image: service-data.docker_image
+            docker-image: service-data['docker-image']
 
     @_setup-services ~>
-      @_get-dependencies-docker-config!
-      @_get-service-docker-config!
-      @_render-docker-compose!
-      @_setup-docker-images (exit-code) ~>
-        | exit-code => @write 'setup failed'; process.exit exit-code
-        @write 'setup complete'
+      @_get-dependencies-docker-config (err) ~>
+        | err => @write 'setup failed'; process.exit
```

Comment:
I think part of this PR makes sense since it added this line. It was added incorrectly at first so I think its worth making sure we add the correct line